parsers/instructions.js

```diff
@@ -5,6 +5,15 @@ import { BlockBlueVerified } from "../shared.js";
 // the response body. since it doesn't match one specific request, it has
 // its own file
 
+const UserTimelineInstructionPath = [
+	"data",
+	"user",
+	"result",
+	"timeline",
```

Comment:
@MrAwesome is the repetition here on purpose?

Reply:
Yes, there was a `timeline` object nested under another `timeline` - I don't remember much, but I remember that being odd. I popped into the dev console to take a quick peek just now, I think they've renamed the outer `timeline` to `timeline_v2` - but I wouldn't trust me too far on that, I'm very very out of the loop. My guess is that a lot of this code is quite stale by this point, especially given all of the... engineering changes over there :(

![image](https://github.com/kheina-com/Blue-Blocker/assets/145945/e4d02324-cfbc-4167-af15-859f0224bd38)

Reply:
Huh weird, on my end I'm still getting `{timeline: timeline: { ... } }`